### PR TITLE
Price in edit and create theme working.

### PR DESCRIPTION
**Pull request by elenicehalma**
The price now is correct. The user inputs 40 and it appears 40 in the theme show.

### CODE REVIEW

File: app/controllers/themes_controller.rb [hunk 1]
```diff
@@ -25,7 +25,6 @@ def new
   def create
     @theme = Theme.new(theme_params)
     @theme.user = current_user
-    @theme.price_cent = @theme.price_cent * 100
```
Comment (vivien-z) [inline app/controllers/themes_controller.rb:28]: Have you confirmed that, if we remove this line, when we create new theme, the price will not be impacted?

Comment (elenicehalma) [reply app/controllers/themes_controller.rb:28]: All is working, now the mathematic calculate is done at the same place. I tested creating and updating.

Comment (vivien-z) [reply app/controllers/themes_controller.rb:28]: nice, will merge it now

Comment (elenicehalma) [reply app/controllers/themes_controller.rb:28]: (price_cente *100) /100

**Pull request merged**